cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -54,11 +57,27 @@ type PolicyProcessor struct {
 	Out                       io.Writer
 }
 
+type PolicyExceptionLister struct {
+	Exceptions []kyvernov2beta1.PolicyException
+}
+
+func (l *PolicyExceptionLister) List(selector labels.Selector) ([]*kyvernov2beta1.PolicyException, error) {
+
+	var matchedExceptions []*kyvernov2beta1.PolicyException
+	for i := range l.Exceptions {
+		matchedExceptions = append(matchedExceptions, &l.Exceptions[i])
+	}
+	return matchedExceptions, nil
+}
+
 func (p *PolicyProcessor) ApplyPoliciesOnResource() ([]engineapi.EngineResponse, error) {
 	cfg := config.NewDefaultConfiguration(false)
 	jp := jmespath.New(cfg)
 	resource := p.Resource
 	namespaceLabels := p.NamespaceSelectorMap[p.Resource.GetNamespace()]
+	policyExceptionLister := &PolicyExceptionLister{
+		Exceptions: p.PolicyExceptions,
+	}
```

Comment:
You have defined a `PolicyExceptionLister` struct that implements the `engineapi.PolicyExceptionSelector` so that you can pass it to the `NewEngine()`.  I don't know if there is a better approach. 
cc @eddycharly

Reply:
not really, we did something similar in the playground